Update arm

KyleR56 opened the pull request:
Updates the Simulator so that it uses the new arm. I only updated the protocol, and did not change the mesh of the arm yet. I forked Connor's C# Server Demo and updated it to work with the new arm if you want to test the changes: https://github.com/KyleR56/ResurgenceServerDemoFork. Just make sure to use the "fix-arm-rendering" branch of Mission Control.

abhaybd reviewed (changes requested):
Since you've tested it I assume everything's good. Minor thing: can you update the readme and replace the wrist differential motors with the wrist?

KyleR56 commented:
I updated the readme with the new joint names. I wasn't able to recreate the bug Victor found where a joint's position is incorrectly reported as 0. To test the bug, I added a print statement that will trigger when a joint's position is reported as 0. When the simulator starts, all joint position reports will trigger the print statement as expected. I then moved every joint via mission control, and the print statements all stopped. As far as I can tell everything works correctly, but if the bug is rediscovered I can take a deeper look.
